_plugins/page-params/page-baseurl-generator.rb

```diff
@@ -6,29 +6,19 @@
 # This custom plugin dynamically sets and injects the page.baseurl variable
 # based on the page's destination.
 #
-# The hook introduces the page.baseurl and
-# page.guide_version data parameters for each page.
+# The hook introduces the page.baseurl parameter for each page.
 # For pages at guides/v2.x the page.baseurl parameter is set
 # as "{site.baseurl}/guides/v#{version}".
 # The {version} is taken from the 'guide_version' front matter parameter on the page;
```

Comment:
Is it still possible to use `guide_version` in the metadata? If not, we should remove this comment.

Reply:
Yes, it's still possible. Front matter parameters always have privilege over default parameters assigned via scopes. (This is how we use layout at the moment.) For more information, refer to [Precedence in Jekyll](https://jekyllrb.com/docs/configuration/front-matter-defaults/#precedence).